src/java.base/share/classes/sun/security/ssl/Finished.java

```diff
@@ -146,7 +146,7 @@ public String toString() {
                     "\"Finished\": '{'\n" +
                     "  \"verify data\": '{'\n" +
                     "{0}\n" +
-                    "  '}'" +
+                    "  '}'\n" +
                     "'}'",
```

Comment:
As you are already here to touch the code, would you like to use text block instead?

Reply:
Could I not change this style?
Maybe another PR should be filed for applying text block feature in sun.security.ssl package.

Reply:
> Could I not change this style?

Sure.  But effort of this update will be overrode if text blocks are used.  I would suggest to use text block once the code block get touched.